R/main.R

```diff
@@ -46,7 +46,7 @@ Options:
 main_worker <- function(args = commandArgs(TRUE)) {
   # nocov start
   args <- main_worker_args(args)
-  wait_for_go_signal(path, args$go_signal)
+  wait_for_go_signal(NULL, args$go_signal)
```

Comment:
surprised about this change, how did this work before?

Reply:
So I think the test cases meant that `path` was never evaluated. As the test `!grepl("^(/|[A-Z][a-z]:)", go_signal) && !is.null(path)` the `!grepl("^(/|[A-Z][a-z]:)", go_signal)` part is FALSE and so `path` never gets used